src/main/webapp/partials/instructorHelpSessions.jsp

```diff
@@ -1018,7 +1018,7 @@
 
       <div class="bs-example">
         <form class="form-horizontal form_question" role="form">
-          <div class="panel panel-primary questionTable" id="questionTable1">
+          <div class="panel panel-primary questionTable" id="intrHelpQuestionTable">
```

Comment:
You cannot use same `id` in the same page. Though it works, but it is not HTML5 compliant.